osu.Game/Rulesets/Edit/HitObjectComposer.cs

```diff
@@ -266,7 +266,7 @@ private void load(OsuConfigManager config, [CanBeNull] Editor editor)
             {
                 item.Selected.DisabledChanged += isDisabled =>
                 {
-                    item.TooltipText = isDisabled ? "Add at least one timing point first!" : ((HitObjectCompositionToolButton)item).TooltipText;
+                    item.TooltipText = isDisabled ? "Add at least one timing point first!" : ((HitObjectCompositionToolButton)item).Tool.TooltipText;
```

Comment:
I think I'd rather see the disabled handling moved inside `HitObjectCompositionToolButton`, rather than what you've done here. It's weird having this handled in this class in the first place.